quickstart/docker/image/ziti-cli-functions.sh

```diff
@@ -1514,27 +1519,32 @@ function setOs {
 }
 
 function checkEnvVariable() {
-
+  _error=false
```

Comment:
if we are assuming bash (or zsh), it would be a good idea to declare this as a `local`.